django_serializer/swagger/utils.py

```diff
@@ -0,0 +1,87 @@
+from django import forms
+from django.forms import ModelForm, modelform_factory
+from marshmallow import Schema, fields
+
+from django_serializer.v2.exceptions import HttpError
+
+FORM_FIELD_MAPPING = {
+    forms.IntegerField: fields.Int,
+    forms.BooleanField: fields.Bool,
+    forms.CharField: fields.Str,
+    forms.DateField: fields.Date,
+    forms.DateTimeField: fields.DateTime,
+    forms.TimeField: fields.Time,
+    forms.DecimalField: fields.Decimal,
+    forms.EmailField: fields.Str,
+    forms.FloatField: fields.Float,
+    # forms.ImageField: ImageField,
+    forms.URLField: fields.Str
+}
```

Comment:
you shoud foresee case with custom fields, example you can find in serializer file